platform/core/Makefile

```diff
@@ -100,13 +101,11 @@ endef
 
 # ============ Android lib ============ #
 
-$(ndk):
-	mkdir -p $(ndk)
-	curl -L $(ndk_download_url) -o $(ndk_archive)
-	unzip -q -d $(ndk) $(ndk_archive)
-	rm $(ndk_archive)
+$(libgcc_paths):
+	# Create a libgcc.a file that links to lunwind for a given ndk architecture.
+	$(os_echo) "INPUT(-lunwind)" >> $@
```

Comment:
Is this related? https://github.com/rust-lang/rust/issues/84040
Anyway, this is needed for debugging, if I got this correctly. Rust will provide these symbols to the compiled binary. This is probably mandatory somehow, right?

Reply:
This is the actual fix -> create a libgcc.a file which redirects to libunwind.

Reply:
Right, but we need either of those for debugging purposes, is that correct?